easybuild/easyconfigs/l/libxc/libxc-2.2.0-intel-2014b.eb

```diff
@@ -8,7 +8,8 @@ description = """Libxc is a library of exchange-correlation functionals for dens
  The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals."""
 
 toolchain = {'name': 'intel', 'version': '2014b'}
-toolchainopts = {'opt': True}
+# Results for some functions (e.g. mgga_c_tpss) change with too aggressive optimization settings.
+toolchainopts = {'opt': False, 'lowopt': True}
```

Comment:
@tovrstra settign `opt` to `False` shouldn't be needed, it's `False` by default, see http://easybuild.readthedocs.io/en/latest/version-specific/toolchain_opts.html#intel

enabling `opt` would imply `-O3`, the default is to use `-O2`, while `lowopt` is equivalent with `-O1` (see also `eb -x libxc.eb | grep CFLAGS`